src/fiatconnect/FiatDetailsScreen.test.tsx

```diff
@@ -326,4 +326,34 @@ describe('FiatDetailsScreen', () => {
     )
     expect(getByTestId('checkmark')).toBeTruthy()
   })
+  it('Mobile info dialog works on mobile money screen', () => {
+    const mmQuote = new FiatConnectQuote({
+      quote: mockFcQuote,
+      fiatAccountType: FiatAccountType.BankAccount,
+      flow: CICOFlow.CashIn,
+    })
+    mmQuote.fiatAccountType = FiatAccountType.MobileMoney
```

Comment:
I can't swap them out and comment out the test because the linter doesn't allow commented out tests. Added comments to remove/replace the code as part of ACT-518.